P/PCRE2/build_tarballs.jl

```diff
@@ -19,13 +21,22 @@ update_configure_scripts
 # Force optimization
 export CFLAGS="${CFLAGS} -O3"
 
-./configure --prefix=$prefix --host=$target --enable-utf8 --enable-unicode-properties --enable-jit
-make -j${nproc} V=1
-make install V=1
+./configure --prefix=${prefix} --host=${target} \
+    --disable-static \
+    --enable-utf \
+    --enable-unicode-properties \
+    --enable-jit \
+    --enable-pcre2-16 \
+    --enable-pcre2-32
+
+make -j${nproc}
+make install
 
 # On windows we need libcpre2-8.dll as well
 if [[ ${target} == *mingw* ]]; then
-    ln -s libpcre2-8-0.dll ${prefix}/bin/libpcre2-8.dll
+    ln -s libpcre2-8-0.dll  ${libdir}/libpcre2-8.dll
+    ln -s libpcre2-16-0.dll ${libdir}/libpcre2-16.dll
+    ln -s libpcre2-32-0.dll ${libdir}/libpcre2-32.dll
```

Comment:
@staticfloat Why are we creating these symlinks?  Because otherwise we we'd drop the `-{8,16,32}` numbers as if they were soversions?  Why creating symlinks instead of simply removing the files?  As it is now, we're creating duplicates in these tarballs.

Reply:
I just followed the pattern from the previous version, which also did this for the 8-bit dll.

Reply:
Yes, I know you just followed the existing step, I was asking Elliott about the rationale :wink:

Reply:
I think the reason is for Julia compatibility.  IMO it's okay to keep on doing this.